winterfell/models/form.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+var _ = require('lodash');
+var FormProps = require('../utils/formProperties');
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var FormSchema = new Schema({
+  _id: String,
+  title: String,                    // Readable title of Form
+  summmary: String,                 // Short summary of Form
+  full_description: String,         // More detailed description of Form
+  fields: [String],                 // Array of "form fields" that Form contains
+  required_fields: [String],        // Array of "form fields" that Form requires
+  required: Boolean                 // Is form required for every claim?
+});
+
+// Query helpers (none)
+// Instance methods (none)
+
+// Static methods
+var allForms = [
+  {
```

Comment:
I dont see the benefit of putting our configs in the db like this, the form configurations should be more accessible. In my other revision I was looking at putting them in static json files that will be accessible to all parts of our app